express-post-json/server.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const arr = [];
+  for (const key in grades) {
+    arr.push(grades[key]);
+  }
+  res.json(arr);
+});
+
+app.use(express.json());
```

Comment:
Typically we would put this at the top of the file, right after creating `app`. It's only required to put it before routes that need it, but it doesn't hurt to have it in there for everything, so typically we put them all at the top.